### PR TITLE
Add a CSV renderer

### PR DESCRIPTION
**Pull request by stevenwilkin**
Is this congruent with the test CSV you've generated for the print house @andrewgarner ?

### CODE REVIEW

File: app/domain/output_document/csv_renderer.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                value_of_pension_pots income_in_retirement continue_working
+                unsure leave_inheritance wants_flexibility wants_security
+                wants_lump_sum poor_health)
+
+    def initialize(output_document)
+      @output_document = output_document
+    end
+
+    def render
+      CSV.generate(col_sep: '|') do |csv|
```
Comment (andrewgarner) [inline app/domain/output_document/csv_renderer.rb]: These are the options I used:
```rb
{ headers: true, col_sep: '|', converters: nil, encoding: 'iso-8859-1:UTF-8' }
```

Comment (stevenwilkin) [inline app/domain/output_document/csv_renderer.rb]: Cool, I'll add in the other options, apart from `headers` as we're only interested in that when we're batching up 1 or more `OutputDocument`s for upload to the print house

Comment (stevenwilkin) [inline app/domain/output_document/csv_renderer.rb]: Looks like `converters` is `nil` by default:

http://ruby-doc.org/stdlib-1.9.2/libdoc/csv/rdoc/CSV.html#DEFAULT_OPTIONS

Comment (andrewgarner) [inline app/domain/output_document/csv_renderer.rb]: :+1:

Comment (stevenwilkin) [inline app/domain/output_document/csv_renderer.rb]: Do you mean `encoding` to be `iso-8859-1`? Ruby complains about `iso-8859-1:UTF-8`:
```
ArgumentError:
       unknown encoding name - iso-8859-1:UTF-8
```

**Pull request closed without merging**